node/silkworm/downloader/internals/working_chain.cpp

```diff
@@ -307,18 +362,18 @@ auto WorkingChain::find_bad_header(const std::vector<BlockHeader>& headers) -> b
 }
 
 auto WorkingChain::accept_headers(const std::vector<BlockHeader>& headers, PeerId peerId)
```

Comment:
Better pass `peerId` by `const &`. And please use `snake_case` for variable names.

Reply:
done! Sorry for the variable names. I have also fixed the WorkingChain data members. I will fix other names in the future.